packages/app/healthcheck-utils/src/HealthcheckResultsStore.ts

```diff
@@ -8,7 +8,7 @@ export type HealthcheckEntry = {
 }
 
 export type HealthcheckResultsStoreParams = {
-  healthcheckNumber: number // how many healthchecks does the system have
+  healthcheckNumber: number // maximum amount of healthchecks that the system can have
```

Comment:
would make sense to change it to `maxHealthcheckNumber` or `maxHealthcheckSupported`, but it would be a breaking change, maybe we can make a not to do it on next major release

Reply:
it wasn't used yet, so maybe we can sneak that one under the radar :D